README.md

```diff
@@ -6,7 +6,31 @@ This is the source code for the [Bitcoin Dev Project](https://bitcoindevs.xyz) t
 
 ## Setup
 
-First, run the development server:
+1) Install the dependencies:
+
+```bash
+npm install
+# or
+yarn install
+# or
+pnpm install
+# or
+bun install
+```
+
+2) Get the `bitcoin-topics` content (initialize the `bitcoin-topics` submodule):
```

Comment:
I recommend removing the submodule name for two reasons:

- To avoid needing to update this file if additional submodules are added in the future.
- The current name is incorrect (refer to #135).

A simplified approach like the following should be enough:
`2) Download submodules`